src/routes/namespaces/[namespace]/workflows/[workflow]/[run]/history/compact/activity-[eventId]/__layout.svelte

```diff
@@ -58,18 +54,16 @@
 <div class="flex flex-col w-full h-full">
   <nav class="mb-4">
     <ul class="flex gap-4 w-full items-start">
-      {#await getEventsGroup(events, params.eventId) then { group, events }}
-        {#each events as event}
-          <li>
-            <a
-              href={getHref(group, event)}
-              class:active={$page.path === getHref(group, event)}
-            >
-              {event.eventType}
-            </a>
-          </li>
-        {/each}
-      {/await}
+      {#each events as event}
+        <li>
+          <a
+            href={getHref(group, event)}
+            class:active={$page.path === getHref(group, event)}
+          >
+            {event.eventType}
+          </a>
+        </li>
+      {/each}
```

Comment:
Might want to poke Candace about an empty state here? I don't think our model _should_ have workflows without events but stranger things have happened.